liferay-frontend/liferay-portal
# LPS-118224 - Remove table and list custom properties

This task removes some components custom properties in order to don’t offer not needed custom properties in classic theme.

---

How to test it:

It does not affect any perceptible component or visual design, but you can inspect HTML and take a look to the list of custom properties, you will be able to see how removed CCPs do not appear

![image](https://user-images.githubusercontent.com/7963804/89189921-fd565a80-d5a0-11ea-8af0-c245e0328c29.png)
